style.css

```diff
@@ -75,25 +113,296 @@ section {
+  gap: 0.5rem;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
+
+.snapshoot-portfolio-icon {
+  border-radius: var(--br-5xs);
+  width: 100%;
+  height: auto;
+  overflow: hidden;
+  object-fit: cover;
+}
+
+.cols img {
+  background: url(blog_drbl_4x.png);
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 4;
+}
+
+.normal-buttonsecondaryenable1 {
+  overflow: hidden;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  position: absolute;
+  top: calc(50% - 24px);
+  left: 0;
+  border-radius: var(--br-5xs);
+  border: 1px solid var(--primary-b400);
+}
+
+.headerTop h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  padding-top: 114px;
+  color: #172b4d;
+  margin-bottom: 0%;
+  z-index: 100;
+}
+
+.headerBottom-bot a {
+  display: flex;
+  flex-direction: column;
+  flex-basis: fit-content;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+  padding-top: 21px;
+  padding-left: 22px;
+}
+
+.tags a {
+  padding: 4px 12px;
+  width: 57px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex-grow: 0;
+  text-decoration-line: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
 .headerBottom a {
+  display: flex;
+  flex-direction: column;
+  flex-basis: fit-content;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
   text-decoration: none;
+  padding-top: 0;
+  padding-left: 6px;
+}
+
+.headerBottom-bot .link-btn a {
+  padding-top: 5px;
+  padding-left: 6px;
+  margin-top: 0;
+}
+
+.headerBottom-bot .link-text {
+  padding-left: 0;
+  margin-bottom: 0;
+}
+
+.tags-bottom {
+  display: flex;
+  flex-direction: column;
+  column-gap: 10rem;
+  flex-wrap: wrap;
+  width: 20%;
+  gap: 10px;
+}
+
+.container.work-area {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-rows: 1;
+  padding-left: 0;
+  padding-top: 0;
+  margin-top: 10px;
+  margin-bottom: 20px;
+
+  .flex-row {
+    margin-left: 0%;
+    padding-left: 16px;
+    padding-top: 10px;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: flex-start;
+    row-gap: 20px;
+    column-gap: 10px;
+
+    .flex-col {
+      /* display: flex;
+      flex-direction: column; */
+      padding-left: 16px;
+      padding-right: 16px;
+      padding-top: 20px;
+      flex-wrap: wrap;
+      width: 327px;
+      height: 550px;
+      background-color: #fff;
+      border: 1px solid #dfe1e6;
+      border-radius: 16px;
+      padding-bottom: 40px;
+    }
+  }
+}
+
+.section-bottom {
+  padding-top: 10px;
+  width: 375px;
+  height: 1263px;
+  left: 375px;
+  top: 4769px;
+  background: #fff;
+  padding-left: 24px;
+  padding-right: 24px;
+  border-top-right-radius: 50px;
+  margin-top: 120px;
+}
```

Comment:
- [ ] As requested by previours reviewer, kindly align contents of this section using flexbox. You can add
```css
    display: flex;
    flex-direction: column;
```

Reply:
The initial reviewer requested the utilization of Flexbox to style the elements in the About section.
The second reviewer, in agreement with the first, made the same request.

- [ ] To meet the project requirements, let's apply Flexbox to arrange the elements in this section.
![image](https://github.com/ethred/Portfolio/assets/83596135/5fc71de3-9666-4b74-8ee9-7be96da81258)